src/Crocodealer/Config.hs

```diff
@@ -0,0 +1,17 @@
+module Crocodealer.Config
+       ( Repo(..)
+       , Config(..)
+       ) where
+
+import Crocodealer.Core.Label
+
+newtype Repo = Repo
+    { unRepo :: Text
+    }
+
+data Config = Config
+    { username            :: Maybe Text
```

Comment:
Could we using the prefix-base naming convention for every field here? Also, we can make all the fields strickt in this data type, like here:

```suggestion
    { configUsername            :: !(Maybe Text)
```